### ISSUE TITLE
Why don’t you merge with gallery-dl which already has wide support for many sites and lot of name recognition,trying to achieve the same: to be YouTube-dL for Images?

### ISSUE THREAD

**Issue by github-userx** [Breizhux/picture-dl#2]
https://github.com/mikf/gallery-dl

Would be great to see you over at the repository! ;) there is even already gallery-get (https://github.com/regosen/gallery_get) but I Gallery-dL Supports the most sites and has the best support in my opinion. I think it’s not a good idea to have so many different projects all in the same programming language (python) trying to achieve the exact Same thing. :)

Kind regards! And it’s great that you got active on GitHub with this!

**Comment by Breizhux** [comment]
> https://github.com/mikf/gallery-dl
> 
> Would be great to see you over at the repository! ;) there is even already gallery-get (https://github.com/regosen/gallery_get) but I Gallery-dL Supports the most sites and has the best support in my opinion. I think it’s not a good idea to have so many different projects all in the same programming language (python) trying to achieve the exact Same thing. :)
> 
> Kind regards! And it’s great that you got active on GitHub with this!

You're absolutely right!
I had done a lot of research because I was surprised that such a tool did not yet exist....
Thank you for the information and encouragement!
This was my first serious project, but I think I'd better continue with you.

Kind regards!

**Comment by github-userx** [comment]
Fantastic! And I hope I didn’t offen you :P I mean the Beauty of opensource is that anyone can fork and also create new and similar projects :)

I just like and prefer when forces are united to create even better pieces of software :) and from my experience gallery-dl is already quite advanced and well written/full featured. 

Feel free to reach out to @mikf if you want to get more involved in the development of gallery-dl or just hop over to the repository :) 

A few months ago, gallery-get still had a few sites that gallery-dl still didn’t support -  but Mike fixed that and created plugins for these missing sites as well and now I am convinced that gallery-dl is the best piece of software when it comes to downloading images [especially from the commandline - I know that for GUI‘s there is still also the legendary Java Programm RipMe: https://github.com/ripmeapp/ripme but I personally prefer commandline anytime! What about You? :) ]

**Comment by Breizhux** [comment]
I'm not offended at all! On the contrary, I would be happy to participate in gallery-dl (within the limits of my abilities). I have browsed the list of supported sites, and I think it would be interesting to add support for PixelFed... I'll look into it;)

**Comment by Breizhux** [comment]
For the interface, I'm pretty sure of the command line (also for youtube-dl). But I also like the idea of developing simple and efficient interfaces that allow you to visually manage image downloading, but I'm not cut out for this kind of project. I prefer to develop a command line database and let others take care of a graphical interface for a computer or a phone for that matter!

**Comment by github-userx** [comment]
That sounds great! PixelFed is the opensource instagram Alternative,isn’t it?

**Comment by Breizhux** [comment]
Yes, of course! But the problem is the decentralization of PixelfFed. I don't know if gallery-dl had already planned this case. I think it took them a while to add PeerTube to youtube-dl because of this problem.... Maybe I'm wrong....
